Sources/Testing/Traits/ConditionTrait+Macro.swift

```diff
@@ -35,8 +35,8 @@ extension Trait where Self == ConditionTrait {
     return "\(platformName) \(version.major).\(minorVersion).\(patchVersion)"
   }
 
-  /// Create a trait controlling availability of a test based on an
-  /// `@available()` attribute applied to it.
+  /// Create a trait that controls a test's availability based on a
```

Comment:
Symbols starting with two underscores are not part of the API surface and don't need revising.